### ISSUE TITLE
Provide a Document Chat Template App

### ISSUE THREAD

**Issue by natdink** [SEMOSS/community#43]
### Is your feature request related to a problem? Please describe.

Not a problem, just a want. It would be nice to have a document chat template app to create broader use cases outside of the policy world. I think this would also attract more users.

### Describe the solution you'd like

Under template apps, I would like to have a Document Chat app that would have all the basic mechanics and code to allow me to upload any files (regardless of subject matter) and start chatting with them...I'm thinking this may result in some prompt re-engineering being necessary. If that's the case, add a statement about it to the app description that we preload in the template.

### Describe alternatives you've considered

N/A

### Additional context
N/a

### Tasks

A list of individual tasks / issues that need to be completed before the ticket is resolved

- [ ] [Create a chat block for the drag and drop ui](https://github.com/SEMOSS/marketplace/issues/11)
- [ ] Templatize it into a reusable app that we show under templates
